--- hostapd-2.6.orig/src/crypto/tls_openssl.c	2015-09-27 20:02:05.000000000 +0100
+++ hostapd-2.6/src/crypto/tls_openssl.c	2016-02-29 21:09:44.355609371 +0000
@@ -38,6 +38,8 @@
   #include "tls.h"
 #include "tls_openssl.h"

+#define OPENSSL_VERSION_NUMBER 0x10001000L
+
 #if !defined(CONFIG_FIPS) &&                             \
     (defined(EAP_FAST) || defined(EAP_FAST_DYNAMIC) ||   \
      defined(EAP_SERVER_FAST))
